app/views/maintenance_tasks/runs/_info.html.erb

```diff
@@ -11,6 +11,14 @@
 
 <% if run.csv_file.present? %>
   <div class="block">
-    <%= link_to('Download CSV', csv_file_download_path(run)) %>
+    <%= link_to("Download CSV", csv_file_download_path(run)) %>
+  </div>
+<% end %>
+<% if run.arguments.present? %>
+  <div class="block">
+    <h5>Arguments:</h5>
+    <% run.arguments.each do |key, value| %>
```

Comment:
Thoughts on dumping a JSON/stringified arguments here? A lot of developers universally understand the syntax of it. And it will reveal a lot of the types that get lost while rendering the value in the DOM (i.e string "0" vs literal 0)
`"{"task_ids": [1, 2 ,3 , 4] }"`

Reply:
This is a great question Andy! The issue with turning args into JSON is we don't know _how_ users are actually using the args they're specifying. Because of the limitations surrounding receiving arguments via a text area, we expect devs who need to use a complex type (like an array / nested hash) to specify it as a string, and then parse it into the data type they want within their Task code. We'd need to infer whether the "12,3,4" in { "task_ids" : "1,2,3,4" } was meant to be turned into an array or left as a string, which we can't really do with the context we have.

On the subject of string "0" vs the numeric 0, Task authors can specify attribute types:
```ruby
attribute :foo, :integer
```
So this can be used to differentiate.